benchflow-test-manager/application/src/main/java/cloud/benchflow/testmanager/tasks/running/DetermineExecuteExperimentsTask.java

```diff
@@ -65,12 +65,10 @@ public void run() {
           selectedExperimentBundle.getExplorationSpaceIndex());
 
       // set experiment as selected
-      explorationModelDAO.addExecutedExplorationPoint(
-          testID,
-          selectedExperimentBundle.getExplorationSpaceIndex()
-      );
+      explorationModelDAO.addExecutedExplorationPoint(testID,
+          selectedExperimentBundle.getExplorationSpaceIndex());
 
-      // save PE defintion to minio
+      // save PE definition to minio
```

Comment:
Done in https://github.com/benchflow/benchflow/pull/394/commits/3169f96597122c1c799e101f6ba8b10b180ebafd